sdk/eng/mgmt/mgmtmetadata/sql_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
```

Comment:
The path of this file should be `eng/mgmt/mgmtmetadata/sql_resource-manager.txt`, rather than under `sdk`

Reply:
Moved, thanks!